src/components/ListPane.scss

```diff
@@ -0,0 +1,31 @@
+.list-pane {
+  background-color: #3e494f;
+  border-collapse: collapse;
+  height: 500px;
+
+  .meta-list {
+    background-color: #272f32;
+    width: 360px;
+  }
+
+  .results-list {
+    display: flex;
+    flex-direction: column;
+    padding: 40px;
```

Comment:
I'm gonna add some design systems globals at some point, but this would be either 32px or 48px.

Reply:
A'ight